library/swift/BUILD

```diff
@@ -46,6 +46,7 @@ swift_library(
         "UpstreamHttpProtocol.swift",
         "filters/*.swift",
         "grpc/*.swift",
+        "ios/*.swift",
```

Comment:
Can you move this to some other directory since UserDefaults aren't iOS-specific? They're available on all apple platforms.

Moving to the bare `library/swift/` directory would be fine.

Reply:
It's not part of OSS swift though right? Is that a distinction we should consider maintaining?

Reply:
I personally think that we should not take into account whether a given Swift API is OSS or not when deciding as to how we structure our code. @jpsim 's suggestion makes sense to me.